ingestion/src/metadata/ingestion/sink/metadata_rest.py

```diff
@@ -129,11 +130,90 @@ def write_record(self, record: Record) -> None:
             self.write_ml_model(record)
         elif isinstance(record, OMetaDatabaseAndModel):
```

Comment:
same here, this will be confusing. Lets add location as the optional field to the existing class

Reply:
Fixed.